components-api/src/main/scala/pl/touk/nussknacker/engine/api/component/ComponentConfig.scala

```diff
@@ -6,7 +6,7 @@ import io.circe.generic.JsonCodec
 import pl.touk.nussknacker.engine.api.definition.{ParameterEditor, ParameterValidator, SimpleParameterEditor}
 import pl.touk.nussknacker.engine.api.parameter.ParameterName
 
-case class SingleComponentConfig(
+case class ComponentConfig(
```

Comment:
Please mention about this change in the migration guide

Reply:
i have aded a coment

Reply:
mentioned